client/src/templates/Introduction/SuperBlockIntro.js

```diff
@@ -143,6 +143,8 @@ class SuperBlockIntroductionPage extends Component {
     const nodesForSuperBlock = edges.map(({ node }) => node);
     const blockDashedNames = uniq(nodesForSuperBlock.map(({ block }) => block));
 
+    // This does not even work, as markdownRemark.frontmatter.superBlock
+    // yields the superBlock Title (non-dashed)???
```

Comment:
We don't use the super block intro markdown any more.  AFAIK, all the intros are in JSON files now.